BangazonAPI/Models/Customer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Customer
+    {
+        public int id { get; set; }
+        public string firstName { get; set; }
+        public string lastName { get; set; }
+
+        public DateTime AccountCreated { get; }
```

Comment:
Let's be consistent between `PascalCase` and `camelCase`-- so far you've been using camel case for the properties on your models (`firstName`, `lastName`, etc) but a few of them (like `AccountCreated`) use pascal case. Discuss as a team and decide on a naming convention and then make sure everything's consistent 😉